nucleus/core/logging/src/main/java/com/sun/enterprise/server/logging/GFFileHandler.java

```diff
@@ -88,53 +101,46 @@
 /**
  * GFFileHandler publishes formatted log Messages to a FILE.
  *
- * @author Jerome Dochez
- * @author Carla Mott
+ * @AUTHOR: Jerome Dochez
+ * @AUTHOR: Carla Mott
```

Comment:
nitpick: could you retain the cleanups from reverted commit, as these invalid javadocs, or typos further down the source?